test/viewmodels/PaymentSearchResultsV2ViewModelSpec.scala

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels
+
+import config.AppConfig
+import models.*
+import org.scalatest.Assertion
+import play.api.Application
+import play.api.i18n.Messages
+import play.twirl.api.HtmlFormat
+import utils.SpecBase
+import utils.TestData.*
+import utils.Utils.*
+import viewmodels.pagination.{ListPaginationViewModel, MetaData}
+
+class PaymentSearchResultsV2ViewModelSpec extends SpecBase {
+
+  "apply method" should {
+
+    "return correct contents without pagination" when {
+
+      "search results are less than records per page in config" in new Setup {
+
+        val viewModel = PaymentSearchResultsViewModel.apply(searchValue = PAYMENT_SEARCH_VALUE,
```

Comment:
Add return type as its a public member. No need to explicitly mention .apply. Better to refactor as below. Same for line 55

        val viewModel: PaymentSearchResultsViewModel =
          PaymentSearchResultsViewModel(
            searchValue = PAYMENT_SEARCH_VALUE,
            account = cashAccount,
            paymentsWithdrawalsAndTransfers = SEQ_OF_PAYMENT_DETAILS_01,
            pageNo = Some(1))